doc_source/constructs.md

```diff
@@ -1,104 +1,100 @@
+
+AWS Construct Library members are found in the `@aws-cdk/aws-NAMESPACE` packages, where NAMESPACE is the short name for the associated service, such as SQS for the AWS Construct Library for the Amazon SQS service\.
 
 ## Construct Structure<a name="constructs_structure"></a>
 
-The construct tree structure is a powerful design pattern for composing high\-level abstractions\. For example, you can define a `StorageLayer` construct that represents your application's storage layer and include all the AWS resources, such as DynamoDB tables and Amazon S3 buckets, needed to implement your storage layer in this construct\. When your higher\-level code uses this construct, it only needs to instantiate the `StorageLayer` construct\.
+Constructs are represented as normal classes in your code and are defined by instantiating an object of that class.
 
-When you initialize a construct, add the construct to the construct tree by specifying the parent construct as the first initializer parameter, an identifier for the construct as the second parameter, and a set of properties for the final parameter, as shown in the following example\.
+When constructs are initialized, they are always defined within the _**scope**_ of another construct and they always have an _**id**_ which must be unique within the same scope.
 
-```
-new SomeConstruct(parent, name[, props]);
+For example, here's how you would define an SNS Topic in your stack with default configuration:
+
+```ts
+new sns.Topic(this, 'MyTopic');
 ```
 
-In almost all cases, you should pass the keyword `this` for the `parent` argument, because you will generally initialize new constructs in the context of the parent construct\. Any descriptive string will do for the `name` argument, and an in\-line object for the set of properties\.
+The first argument to every construct is always the scope in which it is created, and will almost always be simply `this` because most constructs will be defined within the current scope.
 
-```
+Scopes enable constructs to be composed together to form higher-level abstractions by enabling the framework to group them together into logical units, allocate globally unique identifiers and allow them to consult __context information__ such as the region in which it is going to be deployed into, which AZs are available for my account, etc.
+
+In most cases, the construct initializer will have a third `props` argument that can be used to define the construct's initial configuration. For example:
+
+```ts
 new BeautifulConstruct(this, 'Foo', {
-    applicationName: 'myApp',
-    timeout: 300
+  favoriteColor: 'green',
+  timeout: 300
 });
 ```
 
-**Note**  
-Associating the construct to its parent as part of initialization is necessary because the construct occasionally needs contextual information from its parent, such as to which the region the stack is deployed\.
+The `construct.node` property can be used to interact with the construct's node representation:
```

Comment:
You're introducing the concept of a 'node' before explicitly describing the tree.